mlflow/utils/autologging_utils.py

```diff
@@ -9,3 +11,62 @@ def try_mlflow_log(fn, *args, **kwargs):
         fn(*args, **kwargs)
     except Exception as e:  # pylint: disable=broad-except
         warnings.warn("Logging to MLflow failed: " + str(e), stacklevel=2)
+
+
+def param_logger(fn, args=[], kwargs=None, unlogged=[]):
```

Comment:
```suggestion
def log_fn_args_as_params(fn, args=[], kwargs=None, unlogged=[]):
```

Just for clarify of name